README.md

```diff
@@ -1,76 +1,216 @@
+
+```
+ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -J vpcuser@<floating_IP_address> lsfadmin@<management_node_IP_address>
+```
+* where `floating_IP_address` is the floating IP address for the bastion node and `management_node_IP_address` is the IP address for the management node.
+
+* Switch to the LDAP user (for example, switch to lsfuser05):
+
+```
+[lsfadmin@hpccluster-mgmt-1 ~]$ su lsfuser05
+Password:
+[lsfuser05@hpccluster-mgmt-1 lsfadmin]#
+```
+
+* Submit an LSF job as the LDAP user:
+
+```
+[lsfuser05@hpccluster-mgmt-1 lsfadmin]$ bsub -J myjob[1-4] -R "rusage[mem=2G]" sleep 10
+Job <1> is submitted to default queue <normal>.
+```
+
+### Cleaning up the deployed environment:
+
+If you no longer need your deployed IBM Cloud HPC cluster, you can clean it up from your environment. The process is threefold: ensure that the cluster is free of running jobs or working compute nodes, destroy all the associated VPC resources and remove them from your IBM Cloud account, and remove the project from the IBM Cloud console.
+
+**Note**: Ensuring that the cluster is free of running jobs and working compute nodes
+
+Ensure that it is safe to destroy resources:
+
+1. As the `lsfadmin` user, close all LSF queues and kill all jobs:
+   ```
+    badmin qclose all
+    bkill -u all 0
+    ```
+
+2. Wait ten minutes (this is the default idle time), and then check for running jobs:
+    ```
+    bjobs -u all
+    ```
+
+   Look for a `No unfinished job found` message.
+
+
+3. Check that there are no compute nodes (only management nodes should be listed):
+   ```
+    bhosts -w
+   ```
+
+If the cluster has no running jobs or compute nodes, then it is safe to destroy resources from this environment.
+
+#### Destroying resources
+
+1. In the IBM Cloud console, from the **Schematics > Workspaces** view, select **Actions > Destroy resources** > **Confirm** the action by entering the workspace name in the text box and click Destroy to delete all the related VPC resources that were deployed.
+2. If you select the option to destroy resources, decide whether you want to destroy all of them. This action cannot be undone.
+3. Confirm the action by entering the workspace name in the text box and click **Destroy**.
+You can now safely remove the resources from your account.
+
+#### Accessing HPC Tile on Other Regions
+1. If there is a requirement to create HPC cluster other than us-east/us-south/eu-de region, HPC cluster can still be provisioned on another regions.
+2. Instead of using the IBMCloudHPC provider, the automation uses the IBMCloudgen2 providers to spun up the dynamic nodes.
+3. Also instead of using the proxy API URL, a vpc generic API shall be used to spin up the dynamic nodes in the same account as user's.
+4. When creating HPC cluster on different different regions, contract id and cluster id basically not needed. So provide a random contract id and cluster_id.
+
 ## Requirements
```

Comment:
since there is no tf code in the root of the repo, you can delete below sections